tests/mod.rs

```diff
@@ -89,3 +89,8 @@ fn paths_coverage() {
 fn futures_coverage() {
     check_percentage("futures", 1.0f64, true);
 }
+
+#[test]
+fn breaks_expr_coverage() {
+    check_percentage("breaks", 1.0f64, true);
+}
```

Comment:
Missing a newline at the end of the file.

Reply:
Good call, thanks.